admin/sql/updates/20210319-mbs-11451.sql

```diff
@@ -0,0 +1,34 @@
+\set ON_ERROR_STOP 1
+BEGIN;
+
+CREATE TABLE place_meta ( -- replicate
+    id                  INTEGER NOT NULL, -- PK, references place.id CASCADE
+    rating              SMALLINT CHECK (rating >= 0 AND rating <= 100),
+    rating_count        INTEGER
+);
+
+CREATE TABLE place_rating_raw
+(
+    place               INTEGER NOT NULL, -- PK, references place.id
+    editor              INTEGER NOT NULL, -- PK, references editor.id
+    rating              SMALLINT NOT NULL CHECK (rating >= 0 AND rating <= 100)
+);
+
+ALTER TABLE place_meta ADD CONSTRAINT place_meta_pkey PRIMARY KEY (id);
+ALTER TABLE place_rating_raw ADD CONSTRAINT place_rating_raw_pkey PRIMARY KEY (place, editor);
+
+CREATE OR REPLACE FUNCTION a_ins_place() RETURNS trigger AS $$
+BEGIN
+    INSERT INTO place_meta (id) VALUES (NEW.id);
+    RETURN NULL;
+END;
+$$ LANGUAGE 'plpgsql';
+
+INSERT INTO place_meta (id)
+    (SELECT id FROM place)
+    ON CONFLICT (id) DO NOTHING;
```

Comment:
We could leave out `ON CONFLICT (id) DO NOTHING` in this case since the `id` will always be unique (it's the primary key).

```suggestion
INSERT INTO place_meta (id)
    (SELECT id FROM place);
```